# 16 Realized volatility

When 20-print realized volatility sits at or above its pre-2020 80th percentile, is the holdout mean absolute five-print RWTC return larger than on the other days?

Holdout starts 2020-01-01. Verdict: held.

Population standard deviation of the last 20 adjacent RWTC log returns, times the square root of 252. A return enters only when the two prints are 1 to 5 calendar days apart. High realized vol is at or above the train 80th percentile, frozen at 0.4535. Implied volatility is not in this archive. No implied-versus-realized ratio is computed.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'realized-vol')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Train gap in mean absolute five-print return, high vol minus the rest, is 2.618%. Holdout gap is 3.946% (383 high, 1295 rest). The holdout gap is positive, which was the bar.
Mean absolute five-print RWTC return when RV20 is at or above the frozen train 80th percentile
Train | 1550 | 6198 | 2.618%
Holdout | 383 | 1295 | 3.946%


## Method

The bar is a positive holdout gap. The train gap is reported and is not used to move the cutoff.

## Result

Train gap in mean absolute five-print return, high vol minus the rest, is 2.618%. Holdout gap is 3.946% (383 high, 1295 rest). The holdout gap is positive, which was the bar.